shared/domains/chelonia/chelonia.js

```diff
@@ -93,12 +108,23 @@ sbp('sbp/selectors/register', {
       return stack
     }
   },
-  'chelonia/configure': function (config: Object) {
+  'chelonia/config': function () {
+    return cloneDeep(this.config)
+  },
+  'chelonia/configure': async function (config: Object) {
     merge(this.config, config)
     // merge will strip the hooks off of config.hooks when merging from the root of the object
     // because they are functions and cloneDeep doesn't clone functions
-    merge(this.config.hooks, config.hooks || {})
+    Object.assign(this.config.hooks, config.hooks || {})
```

Comment:
Like said in the other comment, it's preferable to use `Object.create(null)` for objects that may have user-supplied keys.

Reply:
Resolving, plz reply here if followup needed: https://github.com/okTurtles/group-income/pull/1308#discussion_r906884592